### ISSUE TITLE
MOT15 test set seqinfo files

### ISSUE THREAD

**Issue by danielzgsilva** [ifzhang/FairMOT#189]
Hi,

Do you have seqinfo files for the test set of MOT15? From what I can tell the link in your readme only contains seqinfo files for the training set. Is it possible to run track.py --test_mot15 without these seqinfo files?

Thanks so much

**Comment by ifzhang** [comment]
Oh, I think it still needs the seqinfo files. You can find the resolution and frame rate of MOT15 test set here: https://motchallenge.net/data/2D_MOT_2015/
I write those seqinfo files by hand.
